src/include/enclave/lthread.h

```diff
@@ -137,7 +137,7 @@ struct lthread
     char funcname[64];            /* optional func name */
     struct lthread* lt_join;      /* lthread we want to join on */
     void** lt_exit_ptr;           /* exit ptr for lthread_join */
-    locale_t locale;              /* locale of current lthread */
+    //locale_t locale;              /* locale of current lthread */
```

Comment:
We should also be able to remove `cancelbuf`, `stdio_locks`, `dlerror_buf`, `dlerror_flag`, `err`.  I'm not sure what `dtv` and `dtv_copy` are, we may also be able to remove them.